crypto/mem.c

```diff
@@ -188,14 +188,14 @@ void *CRYPTO_malloc(size_t num, const char *file, int line)
     void *ptr;
 
     INCREMENT(malloc_count);
-    if (malloc_impl != CRYPTO_malloc) {
+    if (ossl_unlikely(malloc_impl != CRYPTO_malloc)) {
```

Comment:
Instrumenting this file isn't good.  It's only _unlikely_ if custom malloc/free calls haven't been installed.  If they have been installed, this will hurt every time.  Better would be to leave it alone and rely on branch prediction to get it right (which it quickly will).